package.json

```diff
@@ -16,13 +18,25 @@
     "url": "https://github.com/RDary/MyRepo/issues"
   },
   "homepage": "https://github.com/RDary/MyRepo#readme",
-  "dependencies": {
+  "devDependencies": {
+    "@types/chai": "^4.3.3",
+    "@types/jest": "^28.1.6",
+    "@types/mocha": "^9.1.1",
+    "@types/node": "^18.6.4",
+    "@typescript-eslint/eslint-plugin": "^5.33.0",
+    "@typescript-eslint/parser": "^5.33.0",
+    "chai": "^4.3.6",
+    "eslint": "^8.21.0",
+    "jest": "^28.1.3",
+    "mocha": "^10.0.0",
+    "nyc": "^15.1.0",
```

Comment:
Why do you need `nyc` package if you don't use it your scripts. Either delete the package or add a script to check the tests coiverage.